### DISCUSSION TITLE
E-Fil+ Proposal: Enterprise Client DataCap allocation enhancements

### DISCUSSION THREAD

**Discussion by kevzak** [filecoin-project/notary-governance#583] [E-Fil+ WG]
For discussion from the details of the main E-Fil+ issue: https://github.com/filecoin-project/notary-governance/issues/559

It is being proposed that DataCap allocations are client centric, not bound by Fil+ rules, and reflect the scope and size of an Enterprise client's onboarding plan. 

3-4 milestones will manage the release of DataCap:
- PoC
- Post PoC
- Within first 12 months
- 4th tranche - uplift in volume

**Comment by kevzak** [comment]
This week, we held discussions about potential enhancements to the Datacap allocations for enterprise size datasets. Based on a survey to the Notary WG and SP's who have applied for the E-Fil+ Pilot: 90% said that the allotted amounts should be increased from the current max of 5PiB request to better match client data sizes.

50% responded that it should follow the plan of the recent modification proposal: https://github.com/filecoin-project/notary-governance/issues/594

Therefore, the current plan for the E-Fil+ Pilot will be to coincide with the recommended 25PiB request /allocation flow. 

Let’s see the impacts this increase request size and allotments have (positive and negative) to data onboarding during the pilot and potentially reassess as needed to better accommodate enterprise datasets.

**Comment by kevzak** [comment]
Hello - I'd like to re-surface this topic regarding allocation enhancements for Enterprise level projects. We had originally discussed it during the pilot planning phase last year, but we're not ready to propose or manage at the time. 

However, now we're looking to test this use case in more detail and would like to introduce a proposal to change the scope of the pilot program.

We would like to pilot NOT having a DataCap ceiling and allow increased request sizes by instead having upfront, realistic assessment of dataset size and number of replicas. 

A client application will have to: 
- provide proof/verify the size of dataset Ex: Client disclosure, contract, article, screenshot from database, (other ideas?)
- number of copies with clear recipients of each copy (miner ID, business, location)

**Comment by Joss-Hua** [comment]
Hi, has the current DC ceiling blocked eFIL' Client? Perhaps the feedback from the existing eFIL' client here will bring us more references.

**Reply by kevzak** [reply]
> Hi, has the current DC ceiling blocked eFIL' Client? Perhaps the feedback from the existing eFIL' client here will bring us more references.

Hi @Joss-Hua that's a good thought. The majority of current E-Fil applications are PoC size, less than 5PiB, as we started small for the pilot.

There are potentially other projects with much larger data sets that we want to start working with moving forward.
